### ISSUE TITLE
loss nan

### ISSUE THREAD

**Issue by huzippm** [codeslake/IFAN#15]
hello Junyong Lee:
Thanks to your great work!
I want to train the code based on the corrected IAC layer, but I have a question about  loss nan when I directly  re-train the model according to your parameters.
Thank you for your help! @codeslake

**Comment by TPZZZ** [comment]
Hi huzippm,

Did you deal with this problem?
I meet the same problem too, when I retrained the IFAN using my own training setting?
Could you share some suggestions to solve this problem, if you successfully retrained the IFAN.

Thank you for your help

**Comment by huzippm** [comment]
hello TPZZZ:
Did you change the network?
I found that the loss would nan after I added the bn layer.

**Comment by TPZZZ** [comment]
No, I did not change the network.
I followed these command to build the IFAN

from models.IFAN_build import IFANet
from IFAN.configs.config_IFAN import get_config
config = get_config()

model = IFANet(config)

Config is default setting.
